docs/libcurl/curl_multi_waitfds.md

```diff
@@ -0,0 +1,109 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel.se>, et al.
+SPDX-License-Identifier: curl
+Title: curl_multi_waitfds
+Section: 3
+Source: libcurl
+See-also:
+  - curl_multi_cleanup (3)
+  - curl_multi_init (3)
+  - curl_multi_perform (3)
+  - curl_multi_poll (3)
+  - curl_multi_wait (3)
```

Comment:
I think curl_multi_fdset (3) should be there, as it is very similar to this function. Maybe remove init and cleanup calls? They don't seem very closely related to this.

I also think that you could add "curl_multi_waitfds (3)" as a see-also in the curl_multi_fdset.md page.